src/Uno.UI/UI/Xaml/Controls/Grid/RowDefinition.cs

```diff
@@ -22,6 +24,21 @@ internal void OnPropertyChanged2(DependencyPropertyChangedEventArgs args)
 			InvalidateDefinition();
 		}
 
+		// New WinUI assigned ContentProperty syntax
+		public virtual object Content
```

Comment:
Where did you find this property? I do not find it in the documentation.

Reply:
Yes, that's right, it's not in WinUI. I will reverse this and make the implementation via XamlObjectBuilder

Reply:
I originally added this in response to this exception that occurs when running the Given_XamlReader.When_Grid_Uses_New_Assigned_ContentProperty_Syntax test.
<img width="1091" alt="image" src="https://user-images.githubusercontent.com/3002068/162750400-dd46da3d-9c34-4af7-a701-1cad58591cd3.png">
This was the easiest way to implement support for this syntax.